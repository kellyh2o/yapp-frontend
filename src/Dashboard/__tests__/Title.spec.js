import { render } from "@testing-library/react";
import React from "react";
import { Provider } from "react-redux";
import configureMockStore from "redux-mock-store";
import Title from "../Title";

describe('Title', () => {
    it("should render", () => {
        const mockStore = configureMockStore();

        const initialState = {
            demoLocation: {
                name: "Hogwarts",
                jumps: [ 
                    {
                        name: "Jump1",
                        events: []
                    },
                    {
                        name: "Jump2",
                        events: []
                    },
                    {
                        name: "Jump3",
                        events: [
                            {
                                riderMass: 22,
                                riderSpeed: 12,
                            }
                        ]
                    }
                ]
            }
        };
        const store = mockStore(initialState);

        const { asFragment } = render(
            <Provider store={store}>
                <Title />
            </Provider>
        );

        expect(asFragment()).toMatchSnapshot();
    })
});

