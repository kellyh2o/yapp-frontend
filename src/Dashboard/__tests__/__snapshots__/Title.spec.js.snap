// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render 1`] = `
<DocumentFragment>
  <h2
    class="MuiTypography-root MuiTypography-h6 MuiTypography-colorPrimary MuiTypography-gutterBottom"
  />
</DocumentFragment>
`;
